src/Kdyby/Facebook/Resource/ResourceLoader.php

```diff
@@ -59,12 +64,13 @@ class ResourceLoader extends Object implements IteratorAggregate, IResourceLoade
 	 * @param string|NULL $method
 	 * @param array $params
 	 */
-	public function __construct(Facebook $facebook, $pathOrParams, $method = NULL, array $params = array())
+	public function __construct(Facebook $facebook, $pathOrParams, $method = NULL, array $params = array(), $accessToken = NULL)
```

Comment:
I'm really confused about this pullrequest. If you need to pass custom access_token, just do it using the params. Or is there other problem I don't see?